module/zfs/spa.c

```diff
@@ -6432,13 +6523,49 @@ spa_export_common(const char *pool, int new_state, nvlist_t **oldconfig,
 	}
 	spa->spa_is_exporting = B_TRUE;
 
+	/* XXX Should this be chained instead of rejected? */
```

Comment:
It looks like this case is already prevented by the `spa_is_exporting` check above.

Reply:
Done.